### PR TITLE
Switch default CGroup manager to systemd

### PR DESCRIPTION
**Pull request by mheon**
As the name says. CI will be forced to cgroupfs because of a lack of systemd in a container for testing.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: test/e2e/libpod_suite_test.go [hunk 1]
```diff
@@ -141,6 +143,10 @@ func PodmanCreate(tempDir string) PodmanTest {
 	if os.Getenv("STORAGE_OPTIONS") != "" {
 		storageOptions = os.Getenv("STORAGE_OPTIONS")
 	}
+	cgroupManager := CGROUP_MANAGER
+	if os.Getenv("CGROUP_MANAGER") != "" {
+		storageOptions = os.Getenv("CGROUP_MANAGER")
```
Comment (TomSweeneyRedHat) [inline test/e2e/libpod_suite_test.go]: should this be cgroupManager and not storageOptions?

**Comment by TomSweeneyRedHat** [comment]
Looks like all kinds of test unhappiness.

File: test/e2e/libpod_suite_test.go (hunk 1 above)
Comment (mheon) [reply test/e2e/libpod_suite_test.go]: Good catch, but that doesn't appear to be the root of our issues

**Comment by mheon** [comment]
I think i found the test issue - fingers crossed

**Comment by mheon** [comment]
It got past the python tests on my local machine - promising!

**Comment by mheon** [comment]
Alright, I'm going to trigger Homu and see if things pass.
@rh-atomic-bot r+

**Comment by mheon** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@mheon We need to wait for this patch to get merged before we cut a release.

**Comment by mheon** [comment]
@rhatdan Ack - working on it. Hopefully get the tests passing in a few hours.

**Comment by mheon** [comment]
@rhatdan Once this is in - how do you want to handle tagging? We already have a 0.8.2 commit merged, want me to tag that? This will be after it, though

**Comment by baude** [comment]
we can tag a 0.8.2.1 and so on

**Comment by baude** [comment]
@mheon can you check your email for a question I sent you about locking?

**Comment by mheon** [comment]
Everything is passing locally...
@rh-atomic-bot r+

**Comment by mheon** [comment]
Homu and Papr appear to have joined Travis in refusing to run tests.

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
Yes lets just go to a minor number.
8.2.1

**Comment by mheon** [comment]
Ack.
@rh-atomic-bot retry

**Comment by mheon** [comment]
Travis is green on the Homu merge...

I'm going to give this another hour, then merge manually.

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by mheon** [comment]
Tests are not cooperating. Merging manually.

**Pull request merged**